### DISCUSSION TITLE
How to convert tex files into qmd using Pandoc?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#2991] [Q&A]
I am used to transform my tex files to markdown using:

` find ./ -iname "*.tex" -type f -exec sh -c 'pandoc "${0}" -t markdown --wrap=none --reference-links -o "${0%.tex}.md"' {} \;`

The markdown generated is almost perfect for Quarto, but:
- I still need to rewrite all the references to images, sections, etc (footnotes works fine)
- It sometimes skips parts of the tex file and does not show in the output markdown it did so (I wish it wrote in the md file a comment with the tex it did not understand)

I [looked for](https://pandoc.org/MANUAL.html) a qmd output format in Pandoc but couldn't find.  

I wonder if there are best practices for transforming a latex project into a Quarto one.

**Comment by harrelfe** [comment]
The `pandoc` approach is probably the best but see the following for how I used R code and regular expressions to convert my own idiosyncratic usage of $\LaTeX$ to `Quarto`: https://github.com/harrelfe/Rscripts - see the 2 scripts whose names start with `convert`.
